spring-cloud-dataflow-admin-starter/src/main/java/org/springframework/cloud/dataflow/admin/controller/SecurityController.java

```diff
@@ -31,10 +32,11 @@
  */
 @RestController
 @RequestMapping("/security/info")
+@EnableConfigurationProperties(SecurityProperties.class)
 public class SecurityController {
 
 	@Autowired
-	private Environment environment;
+	SecurityProperties securityProperties;
```

Comment:
Sure, this needs to be private. I can also change it to use constructor level injection. Earlier I followed the typical approach to autowire the configuration properties via field injection.